backend/de.metas.frontend-testing/src/main/java/de/metas/frontend_testing/masterdata/mobile_configuration/JsonMobileConfigRequest.java

```diff
@@ -50,6 +50,14 @@ public static class Customer
 		{
 			@NonNull Identifier customer;
 		}
+
+		@Value
+		@Builder
+		@Jacksonized
+		public static class PickingLineConfig
```

Comment:
i think we no longer have the "line" concept, so IMHO this class might be pointless